config/routes.rb

```diff
@@ -18,4 +18,15 @@
   post '/login', to: 'sessions#create'#, as: 'sessions'
 
   delete '/logout', to: 'sessions#destroy'
+
+  namespace :admin do
+    get '/', to: 'dashboard#index'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.